docs/usage.md

```diff
@@ -190,3 +192,178 @@ For example:
 
 Optional childs of the component. This should be a react element that will be
 placed after the input and before the dropdown selection.
+
+<a name="componentclasses"></a>
+### componentClasses
+
+Optional prop that let you stylize your component by adding classes to the
+rendered HTML.
+
+`componentClasses` should be an object containing as a key one of the available
+selectors and as the value an array of strings or a single string containing the
+classes that you want to add to the selector. Since *[classname](https://www.npmjs.com/package/classnames)*
+is available to use the value can be anything that can be accepted by *classname*.
+For Example:
+
+```javascript
+{
+  'wrapper': 'class',
+  'input': 'class1 class2',
+  'token': ['class1 class2', 'myClass', {'otherClass': true}],
+  'facet': [{'myClass': true, 'otherClass': false}]
+}
+```
+
+The available selectors are:
+
+* `wrapper`: the `div` that contains the component
+* `input`: the `input` element of the component
+* `tokenWrapper`: the `div` that contains all the tokens
+* `token`: `span` element that represent the whole token
+* `facet`: `span` element inside the `token` that represent the facet of the
+  token
+* `description`: `span` element inside the `token` that represent the desription
+  of the token
+* `dropdownWrap`: the `div` element that contains the dropdown menu in the facet
+* `dropdownUl`: the `ul` list for the dropdown menu in the facet
+* `dropdownLi`: the `li` elements of the dropdown list
+* `dropdownA`: the `a` elements inside the `li` elements for the dropdown
+* `suggestionsWrap`: the `div` containing the dropdown for the suggestions
+* `suggestionsUl`: the `ul` list for a section of the suggestions dropdown
+* `suggestionsLi`: the `li` elements of the suggestions dropdown list
+* `sectionTitle`: the first `li` for the suggestions dropdown is a section title
+* `suggestionsA`: the `a` elements inside the `li` elements for the suggestions
+  dropdown
+
+If you don't want to add this prop, some of the selectors already have classes
+for you to use in your css:
+
+* `wrapper`: `compound-input`
+* `input`: `compound-input-field`
+* `tokenWrapper`: `token-container`, `selected` (if is selected)
+* `token`: `token`, `facet` (if the facet is shown)
+* `facet`: `facet-type`
+* `description`: `facet-value`
+* `dropdownWrap`: `dropdown`, `token-dropdown`
+* `dropdownLi`: `active` (if is selected)
+* `suggestionsWrap`: `dropdown`, `input-dropdown`
+* `suggestionsLi`: `active` (if is selected)
+* `sectionTitle`: `header`
+
+For ease of use we will write down an example of the rendered structure:
+
```

Comment:
I included this structure to help visualize the final component structure to help the user to know how his changes will affect the component before doing it.